src/main/java/com/spanish_inquisition/battleship/server/MessageBus.java

```diff
@@ -20,6 +23,62 @@ public void addMessage(int senderId, int recipientId, String message) {
         messageBus.add(newMessage);
     }
 
+    public void addResponseHitMessage(int senderId, int recipientId, Integer field) {
```

Comment:
Move all this methods to another class, because it's not responsibility of MessageBus to create them.
Also there is a lot of code duplication.